Dockerfile

```diff
@@ -1,22 +1,17 @@
-FROM golang:1.13.5-alpine as build
+FROM golang:1.14.4-alpine3.12 as build
 
-RUN apk --no-cache add git curl
-RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+WORKDIR /root
+COPY . .
 
-RUN addgroup -g 10001 app && adduser -D -g '' -G app -s /bin/false -h /go/src/github.com/Telefonica/kube-graffiti -u 10001 app
-WORKDIR /go/src/github.com/Telefonica/kube-graffiti
-
-COPY . /go/src/github.com/Telefonica/kube-graffiti
-RUN dep ensure -v
```

Comment:
switch to gomodules

dependencies are fetched automatically on any go command (run, test, build, etc...)